sdk/servicebus/service-bus/samples/v7/typescript/README.md

```diff
@@ -0,0 +1,102 @@
+| [browseMessages.ts][browsemessages]                                 | Demonstrates how to browse a Service Bus message                                                                                                               |
+| [receiveMessagesStreaming.ts][receivemessagesstreaming]             | Demonstrates how to receive Service Bus messages in a stream                                                                                                   |
+| [scheduledMessages.ts][scheduledmessages]                           | Demonstrates how to schedule messages to appear on a Service Bus Queue/Subscription at a later time                                                            |
+| [session.ts][session]                                               | Demonstrates how to send/receive messages to/from session enabled queues/subscriptions in Service Bus                                                          |
+| [useProxy.ts][useproxy]                                             | This sample demonstrates how to create a ServiceBusClient meant to be used in an environment where outgoing network requests have to go through a proxy server |
+| [usingAadAuth.ts][usingaadauth]                                     | This sample how to create a namespace using AAD token credentials                                                                                              |
+| [advanced/administrationClient.ts][advanced_administrationclient]   | Demonstrates how to manage the resources of a service bus namespace.                                                                                           |
+| [advanced/deferral.ts][advanced_deferral]                           | Demonstrates how to defer a message for later processing.                                                                                                      |
+| [advanced/listingEntities.ts][advanced_listingentities]             | Demonstrates how the ServiceBusAdministrationClient can be used to list the entities of a service bus namespace                                                |
+| [advanced/sessionRoundRobin.ts][advanced_sessionroundrobin]         | Demonstrates how to continually read through all the available sessions                                                                                        |
+| [advanced/sessionState.ts][advanced_sessionstate]                   | Demonstrates usage of SessionState.                                                                                                                            |
+| [sendMessages.ts][sendmessages]                                     | Demonstrates how to send messages to Service Bus Queue/Topic                                                                                                   |
```

Comment:
I would like to see the sendMessages in the top.
Can you add more weight for it in the sample?

Reply:
The order I would like to see.
- sendMessages
- receiveMessagesLoop
- receiveMessagesStreaming
- usingAadAuth
- browseMessages
- session.ts
- ... rest of the samples at the root (samples-dev/) ...
- advanced/adminClient.ts
- advanced/sessionRoundRobin.ts
- ... rest ...